Factory/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Factory.AbstractFactory
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            VipAccountFactory vipAccountFactory = new VipAccountFactory("Tom", 20, "King.jpeg");
+            Account vipAccount = vipAccountFactory.Create();
```

Comment:
Factory method should be able to produce different types of object depend on the logic inside

Reply:
Looks like example should be updated with new Factory, right?

Reply:
done